backend/pyladies_about_text

```diff
@@ -0,0 +1,18 @@
+PyLadies - a combination of 2 words: Python and Ladies
```

Comment:
I would put this file in a new folder called `resources`

Reply:
and add an extension to the file, probably txt is ok for now

Reply:
I do not know why I missed the extension :D I meant it to be there :D